app/courses/new/page.module.css

```
@@ -19,9 +19,11 @@
 }
 
 .pageTitle {
+    margin: 0;
     font-family: 'Lato', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Lato" (font-family-name-quotes)

(font-family-name-quotes)